superset/datasets/commands/create.py

```diff
@@ -53,6 +54,9 @@ def run(self) -> Model:
             logger.warning(ex, exc_info=True)
             db.session.rollback()
             raise DatasetCreateFailedError() from ex
+        except SupersetSecurityException as ex:
```

Comment:
@justinpark could you add this to line 53 and also include your message propagation logic.